### PR TITLE
Added helpers to use the post-commit workflow, and still create PRs.

### PR DESCRIPTION
**Pull request by jsuresh**

### CODE REVIEW

File: tools/pr.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+import subprocess
```
Comment (soulplant) [inline tools/pr.py]: This file could do with a comment explaining how to use this script and what it does.

Comment (jsuresh) [inline tools/pr.py]: Done

**Comment by soulplant** [comment]
LGTM

**Comment by jsuresh** [comment]
merged into master

**Pull request merged**